pkgs/top-level/all-packages.nix

```diff
@@ -11298,6 +11298,14 @@ let
 
   cutecom = callPackage ../tools/misc/cutecom { };
 
+  cutegram =
+    let f = ../applications/networking/instant-messengers/telegram;
+        cp = qt5Libs.callPackage;
+    in cp "${f}/cutegram" rec {
```

Comment:
This causes the evaluation error that @domenkozar described. Doing an import of `${f}/cutegram` causes Nix to copy that file to store, which fails in read-only mode.

I also don't understand the need for this non-standard style. Why not just have `libqtelegram-aseman-edition` and `telegram-qml` defined at top-level, like every other package? Or if you really want them to be private to cutegram, just define them inside the cutegram Nix expression, e.g.,
```
let
  libqtelegram-aseman-edition = stdenv.mkDerivation { ... };
  telegram-qml = stdenv.mkDerivation { ... };
in stdenv.mkDerivation {
  name = "cutegram-...";
  buildInputs = [ libqtelegram-aseman-edition telegram-qml ];
};
```